content/fundamentals/module-reference.md

```diff
@@ -162,7 +162,7 @@ export class CatsService {
 
 Manually generated context identifiers (with `ContextIdFactory.create()`) represent DI sub-trees in which `REQUEST` provider is `undefined` as they are not instantiated and managed by the Nest dependency injection system.
 
-To register a custom `REQUEST` object for a manually created DI sub-tree, use the `ModuleRef#registerRequestByContextId()` method, as follows:
```

Comment:
This is correct. A `.` would mean that there is a `static` method on `ModuleRef` called `registerRequestByContextId`. The `#` means there is a method on the _instance_` of `ModuleRef`, but we are referring to `ModuleRef` by its class, not its instance here.